src/Sylius/Behat/Context/Api/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -0,0 +1,622 @@
+    {
+        $this->client->buildUpdateRequest(Resources::PAYMENT_METHODS, $paymentMethod->getCode());
+    }
+
+    /**
+     * @When I name it :name in :language
+     * @When I rename it to :name in :language
+     * @When I remove its name from :language translation
+     */
+    public function iNameItIn(string $language, ?string $name = null): void
+    {
+        $this->client->addRequestData('translations', [$language => ['name' => $name]]);
+    }
+
+    /**
+     * @When I do not name it
+     */
+    public function iDoNotNameIt()
+    {
+        // Intentionally left blank to fulfill context expectation
+    }
+
+    /**
+     * @When I enable it
+     */
+    public function iEnableIt()
+    {
+        $this->client->addRequestData('enabled', true);
+    }
+
+    /**
+     * @When I disable it
+     */
+    public function iDisableIt(): void
+    {
+        $this->client->addRequestData('enabled', false);
+    }
+
+    /**
+     * @When I delete the :paymentMethod payment method
+     * @When I try to delete the :paymentMethod payment method
+     */
+    public function iDeletePaymentMethod(PaymentMethodInterface $paymentMethod)
+    {
+        $this->client->delete(Resources::PAYMENT_METHODS, $paymentMethod->getCode());
+    }
+
+    /**
+     * @Then this payment method :element should be :value
+     */
+    public function thisPaymentMethodElementShouldBe(
+        string $element,
+        string $value,
+    ): void {
+        Assert::true(
+            $this->responseChecker->hasItemWithValue($this->client->index(Resources::PAYMENT_METHODS), $element, $value),
+            sprintf('Payment method should have %s "%s", but it does,', $element, $value),
+        );
+    }
+
+    /**
+     * @When I want to create a new offline payment method
+     * @When I want to create a new payment method with :factory gateway factory
+     */
+    public function iWantToCreateANewPaymentMethod($factory = 'Offline')
+    {
+        $this->client->buildCreateRequest(Resources::PAYMENT_METHODS);
+        $this->client->addRequestData('gatewayConfig', ['factoryName' => $factory, 'gatewayName' => $factory]);
+    }
+
+    /**
+     * @When I specify its code as :code
+     * @When I do not specify its code
+     */
+    public function iSpecifyItsCodeAs($code = null)
+    {
+        $this->client->addRequestData('code', $code);
+    }
+
+    /**
+     * @When I describe it as :description in :language
+     */
+    public function iDescribeItAsIn($description, $language)
+    {
+        $this->client->addRequestData('translations', [$language => ['description' => $description]]);
+    }
+
+    /**
+     * @When make it available in channel :channel
+     */
+    public function iMakeItAvailableInChannel($channel)
+    {
+        $this->client->replaceRequestData('channels', [$this->sectionAwareIriConverter->getIriFromResourceInSection($channel, 'admin')]);
+    }
+
+    /**
+     * @Given I set its instruction as :instructions in :language
```

Comment:
```suggestion
     * @When I set its instruction as :instructions in :localeCode
```
I would be for changing in newly added methods `language` to `localeCode` as it is clearer what it is